### DISCUSSION TITLE
Adding ROIs to existing MPL axes

### DISCUSSION THREAD

**Discussion by liam-stubbington** [jrkerns/pylinac#413] [Q&A] [answered]
Hello, 

I have a single slice DICOM image of a basic image quality phantom. 
I've loaded it as an instance of the DicomImage class (img) and defined several ROIs as DiskROI objects. 

I now want to overlay those ROIs on MPL axes as a quick sanity check, but using the plot2axes method on an existing axes just produces a blank canvas? 

What is the easiest way to do this? 

Thanks so much in advance. 

![image](https://user-images.githubusercontent.com/71460253/138262095-1313f67a-e134-4d9a-9ac1-5e3bc8428e65.png)

**Comment by jrkerns** [comment, marked as answer]
If you want to overlay the ROIs into the original image then you likely want to use the same axis for the ROI and image. I.e.

`fig, ax = plt.subplots()`
`img.plot(ax)`
`north.plot2axes(ax, 'red')`

Let me know how it goes.

**Reply by liam-stubbington** [reply]
Unfortunately the ROI doesn't show up? 
![image](https://user-images.githubusercontent.com/71460253/138308625-a7d294ae-a10e-4c0a-bf31-ab3d811af88e.png)

**Reply by jrkerns** [reply]
Swap the order of the `img.plot()` with the `north.plot...`

**Reply by liam-stubbington** [reply]
Okay, thanks @jrkerns - this has worked
